ext/iconv/iconv.c

```diff
@@ -47,6 +47,34 @@
 #include "zend_smart_str.h"
 #include "ext/standard/base64.h"
 #include "ext/standard/quot_print.h"
+
+#ifdef PHP_ICONV_IMPL
+#define PHP_ICONV_IMPL_VALUE PHP_ICONV_IMPL
+#elif HAVE_LIBICONV
+#define PHP_ICONV_IMPL_VALUE "libiconv"
+#else
+#define PHP_ICONV_IMPL_VALUE "unknown"
+#endif
+
+char *get_iconv_version(void) {
+	char *version = "unknown";
+
+#ifdef HAVE_LIBICONV
+	static char buf[16];
+	snprintf(buf, sizeof(buf), "%d.%d", _libiconv_version >> 8, _libiconv_version & 0xff);
+	version = buf;
+#elif HAVE_GLIBC_ICONV
+	version = (char *) gnu_get_libc_version();
+#endif
+
+	return version;
+}
+
+#define PHP_ICONV_VERSION_VALUE get_iconv_version()
```

Comment:
I would prefer
```suggestion
#define PHP_ICONV_VERSION_VALUE() get_iconv_version()
```
but I suppose that would not easily work for autogeneration. As such, this is okay as is.

Reply:
There shouldn't be a problem with this, but I didn't bother to try it out until now :) I realized that the macro is not even needed.

Reply:
Oh, cool! That's even better.